src/Chainweb/Difficulty.hs

```diff
@@ -338,22 +338,22 @@ window Testnet00 = Just $ WindowWidth 120
 -- 120 blocks, should take 1 hour given a 30 second BlockRate.
 window Testnet01 = Just $ WindowWidth 120
 
--- | The maximum number of bits that a single application of `adjust` can apply
--- to some `HashTarget`. As mentioned in `adjust`, this value should be above
--- \(e = 2.71828\cdots\).
+-- | The minimum factor of change that a single application of `adjust` must
+-- apply to some `HashTarget` for it to be accepted. As mentioned in `adjust`,
+-- this value should be above \(e = 2.71828\cdots\).
 --
-newtype MaxAdjustment = MaxAdjustment Natural
+newtype MinAdjustment = MinAdjustment Natural
 
--- | The Proof-of-Work `MaxAdjustment` for each `ChainwebVersion`. For chainwebs
+-- | The Proof-of-Work `MinAdjustment` for each `ChainwebVersion`. For chainwebs
 -- that do not expect to perform POW, this should be `Nothing`.
 --
-maxAdjust :: ChainwebVersion -> Maybe MaxAdjustment
-maxAdjust Test{} = Nothing
-maxAdjust TestWithTime{} = Nothing
-maxAdjust TestWithPow{} = Just $ MaxAdjustment 3
+minAdjust :: ChainwebVersion -> Maybe MinAdjustment
+minAdjust Test{} = Nothing
+minAdjust TestWithTime{} = Nothing
+minAdjust TestWithPow{} = Just $ MinAdjustment 3
 -- See `adjust` for motivation.
-maxAdjust Testnet00 = Just $ MaxAdjustment 3
-maxAdjust Testnet01 = Just $ MaxAdjustment 3
+minAdjust Testnet00 = Just $ MinAdjustment 3
+minAdjust Testnet01 = Just $ MinAdjustment 3
```

Comment:
We dispatch on `ChainwebVersion` here, instead of hard-coding. This opens the possibility of transparently updating the value for future `ChainwebVersion`s. 

Should we hard-code this instead?

Reply:
LGTM